### ISSUE TITLE
Fixed wordCountTest - Position: 252924 > readLimit: 236096

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Map#381]
see https://teamcity.chronicle.software/viewLog.html?buildId=642243&buildTypeId=OpenHFT_BuildAll_BuildJava8compileJava8

```
19:24
this word count test has just happened again - see https://teamcity.chronicle.software/viewLog.html?buildId=642243&buildTypeId=OpenHFT_BuildAll_BuildJava8compileJava8
19:24
First failure: refs/heads/main
#3708  No changes 06 Jan 22 19:55
net.openhft.chronicle.bytes.util.DecoratedBufferUnderflowException: readPosition failed. Position: 252924 > readLimit: 236096
	at net.openhft.chronicle.bytes.AbstractBytes.readPosition(AbstractBytes.java:252)
	at net.openhft.chronicle.bytes.AbstractBytes.readPosition(AbstractBytes.java:53)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.initKeySearch(CompiledMapQueryContext.java:3836)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.searchState(CompiledMapQueryContext.java:3856)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.searchStatePresent(CompiledMapQueryContext.java:3878)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.initPresenceOfEntry(CompiledMapQueryContext.java:3926)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.entryPresence(CompiledMapQueryContext.java:3935)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.entryPresent(CompiledMapQueryContext.java:3945)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.entry(CompiledMapQueryContext.java:4061)
	at net.openhft.chronicle.map.impl.CompiledMapQueryContext.entry(CompiledMapQueryContext.java:40)
	at net.openhft.chronicle.map.MapMethodsSupport.tryReturnCurrentValueIfPresent(MapMethodsSupport.java:34)
	at net.openhft.chronicle.map.MapMethods.acquireUsing(MapMethods.java:152)
	at net.openhft.chronicle.map.VanillaChronicleMap.acquireUsingBody(VanillaChronicleMap.java:815)
	at net.openhft.chronicle.map.VanillaChronicleMap.acquireContext(VanillaChronicleMap.java:357)
	at net.openhft.chronicle.map.VanillaChronicleMap.acquireContext(VanillaChronicleMap.java:61)
	at eg.WordCountTest.wordCountTest(WordCountTest.java:73)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
```

Send a message to Per Minborg

**Comment by RobAustin** [comment]
@minborg plese retest and if fixed, close.

**Comment by minborg** [comment]
I ran it locally over 200 times with no failure. So, closing

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.23ea3](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.23ea3), [BOM-2.23ea96](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea96)
